modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -5143,17 +5210,16 @@ RETURN *
 
 Description of the returned code::
 Variable in subquery is shadowing a variable with the same name from the outer scope.
-If you want to use that variable instead, it must be imported into the subquery using importing `WITH` clause. (the shadowing variable is: `n`)
+If you want to use that variable instead, it must be imported into the subquery using a variable scope clause. (the shadowing variable is: `n`)
```

Comment:
This is the Neo4j code. I thought that changing the message was a breaking change.